wordpress-mobile/WordPress-iOS
# Comment entity: set default values on attributes

Ref: #16876

This sets default values on the `Comment` model attributes. 
- The attributes are still optional in Core Data.
- For `Int`s and `Bool`s, removed setting default values in `CommentService`.
- For optional `String`s, the properties in `Comment+CoreDataProperties` are now required.

To test:

My Site > Comments:
- Comment statuses are correct (in the correct filters).
- Comment content is displayed correctly.
  - Author info
  - Post title
  - Body
- Edit a comment. Content is displayed correctly.
- Comments correctly reflect your Like status.
- Long press on a Comment date. Verify `Copy Link to Comment` copies the URL.
- Toggle the `unifiedCommentsAndNotificationsList` feature. Verify the Comments list date sections are correct.
- Repeat the steps above on a self-hosted site.

Reader posts:
- Comment content is displayed correctly.
- Comments correctly reflect your Like status.
- Reader post comments hierarchy is correct.

## Regression Notes
1. Potential unintended areas of impact
Comment content could be displayed incorrectly.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested functionality and compared with previous app version.

3. What automated tests I added (or what prevented me from doing so)
I'll address in a future PR.

PR submission checklist:
